node_normalizer/server.py

```diff
@@ -407,20 +407,22 @@ async def get_curie_prefixes_handler(
     # these supresses such warnings.
     logging.captureWarnings(capture=True)
     warnings.filterwarnings("ignore", category=ResourceWarning)
-    plater_service_name = os.environ.get('SERVER_NAME', 'infores:sri-node-normalizer')
-    assert plater_service_name and isinstance(plater_service_name, str)
 
-    jaeger_exporter = JaegerExporter(
-        agent_host_name=os.environ.get("JAEGER_HOST", "localhost"),
-        agent_port=int(os.environ.get("JAEGER_PORT", "6831")),
-    )
-    resource = Resource(attributes={
-        SERVICE_NAME: os.environ.get("JAEGER_SERVICE_NAME", plater_service_name),
+    otel_service_name = os.environ.get('SERVER_NAME', 'infores:sri-node-normalizer')
+    assert otel_service_name and isinstance(otel_service_name, str)
+
+    otlp_host = os.environ.get("JAEGER_HOST", "http://localhost/").rstrip('/')
```

Comment:
Is the `http://` correct here? The example of the JAEGER_HOST you sent me on Slack does not have any prefix.

Reply:
Good catch, yes, the http is correct and seems to be necessary (though I'm not sure why).